<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RegistrationWindow</class>
    <widget class="QWidget" name="RegistrationWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>845</width>
                <height>497</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="QFrame" name="frame">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>60</y>
                    <width>691</width>
                    <height>341</height>
                </rect>
            </property>
            <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
                <enum>QFrame::Raised</enum>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>691</width>
                        <height>351</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <widget class="Line" name="line_3">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_5">
                            <item>
                                <widget class="QLabel" name="label_4">
                                    <property name="text">
                                        <string>用户名:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line_8">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="user_name_line_edit"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="Line" name="line_9">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>账号：</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="user_id_line_edit"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="Line" name="line_5">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="text">
                                        <string>密码：</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line_2">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="user_password_line_edit"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="Line" name="line_6">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <item>
                                <widget class="QPushButton" name="registration_button">
                                    <property name="text">
                                        <string>注册</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line_7">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="exit_button">
                                    <property name="text">
                                        <string>返回到登入界面</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="Line" name="line_4">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
